script/Makefile

```diff
@@ -246,55 +246,60 @@ test: do-build
 # Note
 # This should execute all tests and not exit if one of the suites fail
 #
-test-integration: do-build
+test-integration: do-build gotestfmt-install
```

Comment:
Why not to put `gotestfmt-install`as part of the `do-build` procedure?

Reply:
It was because `gotestfmt-install` is used only for testing. But agreed that by putting it to `do-build` will make Makefile less cluttered. I'll change it accordingly.